install.sh

```diff
@@ -517,6 +518,9 @@ while true; do
         --legacy_hipblas_direct)
             legacy_hipblas_direct=true
             shift ;;
+        --dev_logic_filter|--dev-logic-filter)
+            logic_filter=${2}
+            shift 2;;
         --) shift ; break ;;
```

Comment:
Oh, I think we can update the option description in --help messages

Reply:
Will amend this.

Reply:
just a recommendation: revise the naming as "logic_yaml_filter"